src/lib/error/ErrorSection.svelte

```diff
@@ -0,0 +1,8 @@
+<script lang="ts">
+    export let side: string, heading: string, text: string;
```

Comment:
```suggestion
    export let align: string, heading: string, text: string;
```

Align seems like a better name and it's consistent with tailwind https://tailwindcss.com/docs/text-align#setting-the-text-alignment